compiler/src/dotty/tools/backend/jvm/BCodeSkelBuilder.scala

```diff
@@ -147,7 +174,7 @@ trait BCodeSkelBuilder extends BCodeHelpers {
         case _ => ()
       }
 
-      val ssa = getAnnotPickle(thisName, claszSymbol)
+      val ssa = None // getAnnotPickle(thisName, claszSymbol)
```

Comment:
Why is this commented out?

Reply:
The operation was inlined. Kept the old code to give some contexts of what it was used.

Reply:
Added TODO comment